### ISSUE TITLE
License Discussion

### ISSUE THREAD

**Issue by jasmineortega** [UBC-MDS/imghelp#59]
Link to more in-depth license issue discussion for ImgHelp/ImgHelpR: https://github.com/UBC-MDS/ImgHelpR/issues/32

--
Hi everyone -- here is the issue where we should discuss the licensing choice for our packages (ImgHelp/ImgHelpR) as discussed in Milestone 4. For both packages, we previously chose to use the MIT license. How does everyone feel about the license as it currently is? For more in depth details about our options, please refer to [lecture 8](https://pages.github.ubc.ca/mds-2021-22/DSCI_524_collab-sw-dev_students/materials/lectures/08_lecture-who-owns-the-code-and-how-can-I-use-it.html)

As is, the MIT license seems pretty flexible in terms of derivative work/collaboration, which is okay with me. I believe our intention of creating a beginner-friendly image package goes hand in hand with a more lax/open source type of license that encourages others to build upon our work!

**Comment by stevenlio88** [comment]
I have the same opinion about this package as well. This is meant for learning purposes and anyone is free to contribute and use this package at their own risk.

**Comment by MaeveShi** [comment]
Same here, anyone will be able to use our package and also give feedbacks, to help us improve the package.

**Comment by jasmineortega** [comment]
I agree that the future of this package would work best under a flexible license that allows other to collaborate or expand on the existing framework we've made. I especially like that the MIT license provides a liability protection for us as well, which I feel like is a "better safe than sorry" protection we could use. 

It sounds like are all in agreement about the MIT license. We will go ahead and keep the license as is then!
